db/dbformat.h

```diff
@@ -518,7 +540,7 @@ class IterKey {
 
   bool IsKeyPinned() const { return (key_ != buf_); }
 
-  // user_key does not have timestamp.
+  // ts is appended to user_key if provided.
```

Comment:
Does 'user_key' have timestamp?

Reply:
It should not when `ts` is provided. Updated comment.